components/script/dom/webglrenderingcontext.rs

```diff
@@ -1688,14 +1688,19 @@ impl WebGLRenderingContext {
                 constants::BOOL
                 | constants::INT
                 | constants::SAMPLER_2D
+                | WebGL2RenderingContextConstants::SAMPLER_2D_ARRAY
```

Comment:
We'll need to be more exact with the new match values in this file: https://github.com/servo/servo/blob/58ba61d3d2185c4296fb363dfbf869c9d98df18b/components/script/dom/webglrenderingcontext.rs#L1037

Reply:
I've added it in #37738 as follow up issue.